docs/guide/generate_objects_doc.py

```diff
@@ -62,7 +62,7 @@
 fileList = sorted(fileList)
 
 # create the 'appearances' page
-with open('appearances.md', 'w', encoding='utf-8') as file:
+with open('appearances.md', 'w', encoding='utf-8', newline='\n') as file:
```

Comment:
> Lines in the input can end in '\n', '\r', or '\r\n', and these are translated into '\n' before being returned to the caller.

https://docs.python.org/3/library/functions.html#open

Why does the default value for the new line makes a problem on Windows. Furthermore, on Windows, the new line should be detonated with `\r\n`?

Reply:
On Windows, the default is `\r\n` which doesn't match with the files committed which were generated under Linux. For the committed file, we use the Linux standard, hence the `newline='\n'` enforcement for Windows.